robot_smach_states/src/robot_smach_states/manipulation/grab.py

```diff
@@ -80,13 +78,16 @@ class PickUp(smach.State):
     REQUIRED_ARM_PROPERTIES = {"required_gripper_types": [GripperTypes.GRASPING],
                                "required_goals": ["carrying_pose"], }
 
-    def __init__(self, robot, arm, grab_entity, check_occupancy=False):
+    def __init__(self, robot: Robot, arm: ArmDesignator, grab_entity: Designator,
+                 check_occupancy: bool = False, visual_servoing: bool = False) -> None:
```

Comment:
At some point we need to think about how to manage these auxilary functions. We cant keep adding flags to turn stuff on or off.
But for now this is alright. In the future we may need to think about adding a pre-hook and a post-hook to be more general.

Reply:
If we end up removing the visual servoing check below in lines 181-188, we can delete the flag